test/data/javascript/tests.js

```diff
@@ -2490,6 +2490,20 @@ exports.test_data = {
             '    </{a + b}>',
             ');'
           ]
+        },
+        {
+          unchanged: [
+            'class Columns extends React.Component {',
+            '    render() {',
+            '        return (',
+            '            <a>',
+            '              <td>Hello</td>',
+            '              <td>World</td>',
+            '            </a>',
```

Comment:
This needs to be empty to actually test your change right? 
```suggestion
            '            <>',
            '              <td>Hello</td>',
            '              <td>World</td>',
            '            </>',
```

Reply:
Oh! I didn't see I hadn't changed it back, I will change this now